calico_versioned_docs/version-3.27/network-policy/get-started/kubernetes-default-deny.mdx

```diff
@@ -33,9 +33,13 @@ For compatibility with Kubernetes, **{{prodname}} network policy** enforcement f
 
 For other endpoint types (VMs, host interfaces), the default behavior is to deny traffic. Only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
 
-### Best practice: implicit default deny policy
+### Best practice: create a default deny policy
 
-We recommend creating an implicit default deny policy for your Kubernetes pods, regardless of whether you use {{prodname}} or Kubernetes network policy. This ensures that unwanted traffic is denied by default. Note that implicit default deny policy always occurs last; if any other policy allows the traffic, then the deny does not come into effect. The deny is executed only after all other policies are evaluated.
+We recommend that you create a {{prodname}} default deny policy (network policy with a `Deny` rule) for your Kubernetes pods shortly after installing {{prodname}}. This policy prevents pods without policy, incorrect policy, empty policy, and {{prodname}} policy for non-workload endpoints, from allowing traffic until you can put policies in place and test them. After you write policy for the traffic that you want to allow in the cluster, we recommend that you create a *global default deny policy* to secure your deployment. In short, creating a default deny policy covers security gaps without you having to remember default deny/allow behavior of Kubernetes and {{prodname}} policies.
```

Comment:
The example below doesn't agree with this.  Here it says "with a `Deny` rule" but below, there is no Deny rule.  The policy below has no rules at all!

That's because the policy below is only added for its side effect... The default policy action (allow/deny) is:

* Allow, if your workload has no policy at all.
* Deny, if your policy has any policy at all.

Adding an empty policy has the side effect of triggering the above "Deny, if your policy has any policy at all" but the policy has no rules in it, so it doesn't matter what order it has because no packets can ever match the policy, only its side-effect.